Update pages about links for content and for design (#314)
wpaccessibility/wp-a11y-docs: docs/topics/content/links/link-image.md

parent: Links
description: For linked images, the alt attribute (the alternative text) can be used as the link text.
nav_order: 3
contributors:
  - Joe Dolson
  - Rian Rietveld
---

# Images as links

> For linked images, the `alt` attribute (the alternative text) can be used as the link text. Screen readers will announce the alt text when the link gets focus.

Reviewer: ```suggestion
For linked images, the `alt` attribute (the alternative text) is used as the link text. Screen readers will announce the alt text when the link gets focus.
```

> But: if the alt attribute describes the image, the link text will be the description of the image, which is unlikely to clearly communicate the link purpose.

Reviewer: ```suggestion
If the alt attribute describes the image, the link text will be the description of the image, which is unlikely to clearly communicate the link purpose.
```

> It is not necessary to ever use the word ‘link’ or 'this link goes to' in your alt text; that will already be sufficient announced by the screen reader.

Reviewer: ```suggestion
It is not helpful to use the word ‘link’ or 'this link goes to' in your alt text; that will already be announced by the screen reader.
```

{: .callout .dont }

> **Don't**: leave the alt text out for an image that is used as link.    

Reviewer: ```suggestion
**Don't**: omit alt text for an image that is used as link.    
```

<a href="{{site.baseurl}}"><img src="{{site.baseurl}}/assets/images/logo-wa11ypuu.png" alt=""></a>

> The screen reader VoiceOver announces this as: "**Link, link image**". So there is no info at all about where the link leads to.

Reviewer: ```suggestion
The screen reader VoiceOver announces this as: "**Link, link image**". So there is no information about where the link leads.
```
